application-admintools-ui/src/main/resources/AdminTools/WebHome.xml

```diff
@@ -453,7 +453,14 @@
   background-color: $theme.pageBackgroundColor;
   border-color: $theme.pageContentBackgroundColor;
   padding: 1rem 1.1rem;
-  min-height: 40rem;
+}
+
+.first-row {
+  min-height: 35rem;
+}
+
+.second-row {
+  min-height: 45rem;
 }
```

Comment:
You have a specific min-height depending on the gadgets / the information there is on a row, right? But this doesn't mean you are assuming that a specific gadget will always be displayed on a specific row? Can't a user modify the gadget position?
Also, these classes are too generic